core/src/main/java/nl/nn/adapterframework/core/IXmlValidator.java

```diff
@@ -14,4 +14,5 @@
 	public String getSchema();
 	public String getSchemaLocation();
 	public Set<XSD> getXsds() throws ConfigurationException;
+	public String getInfo();
```

Comment:
Ik wil niet het interface IXmlValidator aanpassen voor deze functionaliteit, waarvan je zegt dat hij alleen op de AM-API gebruikt wordt, en in de toekomst ook niet gebruikt zal worden, omdat er dan MTOM gedaan wordt,

Reply:
Je schreef:
Ik zou geholpen zijn als bij het genereren van een wsdl vanuit de IBIS-console het comment gedeelte in de wsdl aanpasbaar is (is nu fixed text). Maar daarvoor moet m.i. wel de IXmlValidator aangepast worden. Of heb jij wellicht nog een ander idee hoe dit te fixen?

Laten we de method dan getDocumentation() noemen, en het in IXmlValidator documenteren in Javadoc:
"provide additional generic documentation on the validation or the subsequent processing. This documentation will be included in generated schema's like WSDL or OpenApi"